packages/language-server/test/plugins/typescript/features/CodeActionsProvider.test.ts

```diff
@@ -115,8 +118,7 @@ describe('TypeScript Plugin#CodeActionsProvider', () => {
 						{
 							edits: [
 								{
-									newText:
-										"import Basic from './basic.astro';\n\timport QuickFixImportComponent from './quickFixImportComponent.astro';\n",
+									newText: `import Basic from './basic.astro';${newLine}\timport QuickFixImportComponent from './quickFixImportComponent.astro';${newLine}`,
```

Comment:
Seems like I was 10 seconds to late :sweat_smile:

Reply:
Haha, sorry! It's a valid suggestion still

I think I prefer doing it with a template string because while it match this inserted text nicely, it doesn't necessarily for others and I'd rather keep the tests consistent. Sort / organize imports and auto-import with no frontmatter are really the only two actions that does this kind of multi-line insert